history/titles/00_k_stormwind.txt

```diff
@@ -315,33 +315,22 @@ c_azora = {
 	652.7.23 = { holder = azerothian14 } #Osion[18]
 }
 c_stonecairn = {
-	1.1.1={
-		liege = k_stormwind
-	}
-	26.5.7={holder=1011} #Landan[16110]
-	108.8.18={holder=1016} #Aedis[16110]
-	120.11.7={holder=1022} #Kristoff[16110]
-	182.6.9={holder=1024} #Bazzil[16110]
-	193.2.8={holder=1031} #Gaiman[16110]
-	238.12.11={holder=1038} #Sullivan[16110]
-	280.11.1={holder=1043} #Philip[16110]
-	320.1.1={holder=1045} #Bert[16110]
-	329.6.17={holder=1048} #Tito[16110]
-	359.1.25={holder=1052} #Carrington[16110]
-	396.3.24={holder=1063} #Barrett[]
-	432.6.23={holder=1066} #Nico[]
-	473.8.23={holder=1075} #Bruce[]
-	513.8.27={holder=1077} #Parr[]
-	541.9.5={holder=1084} #Carson[]
-	542.5.30={holder=261} #Anson[3]
-	554.04.18={holder=263} #Brentan[3]
-	557.12.03={holder=267} #Augustus[3]
-	# Usurped by patron
-	580.1.1={holder=310240} #Elendur[patron]
+	20.1.1={holder=murloc120}
+	68.4.19={holder=murloc121}
+	135.6.18={holder=murloc122}
+	169.3.21={holder=murloc123}
+	205.6.23={holder=murloc124}
+	243.8.25={holder=murloc125}
+	281.11.27={holder=murloc126}
+	329.11.4={holder=murloc127}
+	396.5.6={holder=murloc128}
+	465.7.8={holder=murloc129}
+	501.7.13={holder=murloc130}
+	539.9.15={holder=murloc131}
 	# Conquered by the Horde
 	587.10.1={holder=10050} #Orgrim[2001]
-	# Retaken by Stormwind
-	590.11.1={holder=310240} #Elendur[patron]
+	# Horde broken, murlocs move back in
+	590.11.2={holder=murloc131}
```

Comment:
`murloc131` dies in `596.3.17` so you need to pass the title to his heir.

Reply:
Added that in.